### ISSUE TITLE
Cannot download .netframework

### ISSUE THREAD

**Issue by Paula368** [dotnet/core#5337]
Problem encountered on https://dotnet.microsoft.com/learn/dotnet/hello-world-tutorial/install
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

I received a 'dotnet' is not recognized as an internal or external command error.  I have used a new command prompt and tried restarting my computer.

**Comment by mairaw** [comment]
The right place to download the .NET Framework would be at https://dotnet.microsoft.com/download/dotnet-framework.

If you want to let us know how you ended up in the tutorial instead, that would help us to improve the experience for our users.

**Comment by Paula368** [comment]
Dear Maria

Thank you for getting back to me.  I was told by work IT Heldesk to google .NETFramework and download it from the website which I did and is how I got onto the tutorial.  Once I downloaded it I then got the error message when I went to the command prompt > dotnet.  That said, today I have not done anything except retried to upload the original files/programme that was requesting the .NET Framework and it has worked……fortunately.

Keeping everything crossed I am now sorted.

Many thanks.

Paula
